todo.md

```diff
@@ -52,59 +37,88 @@
 
 * CTRL+c does not work in windows, all examples do not clean up
 
+* `Directory`, `Path`, `FileName`, `FilePath` cannot handle UTF-8 file names
+  * `Directory` panics when a directory contains non-ascii (UTF-8) characters
```

Comment:
I think the solution would be to use `&str` as parameter when these types are created. This fixes the problem of invalid UTF-8 strings. `str` has a `is_ascii` method which then could be used to ensure that only ascii characters are used.

Reply:
You are right. The `todo.md` file is a deprecated file and I will keep it around until the whole content is moved into other documents or issues.